tests/ci/clean_up.sh

```diff
@@ -15,9 +15,9 @@ HTTPD_PATH=${HTTPD_PATH:-}
 greenprint "🧼 Cleaning up: start"
 # Remove stop and remove the VM
 virsh destroy "${IMAGE_KEY}" || true
-virsh undefine "${IMAGE_KEY}" --nvram
-#TODO: Remove the resulted VM until we find a way to save it at S3
+virsh undefine "${IMAGE_KEY}" --nvram || true
 virsh vol-delete --pool images "${IMAGE_KEY}".qcow2
```

Comment:
@guy9050 Now this line is not needed and it'll fail because there is no `.qcow2` file, so it won't run the next of the script.